### DISCUSSION TITLE
Form value is empty when saving

### DISCUSSION THREAD

**Discussion by feurer98** [Budibase/budibase#6394] [Help] [answered]
Hello together

I am currently building a tool with budibase.
Now it has eigendlich zimlich worked well, only with this form I do not know why the value is not stored.
The value that is in the ANU is calculated, but it does not save anything if I write by hand in the field.

I have attached you pictures, maybe someone sees the error.
It is certain that "Auftragsnummer" must not be zero.

### The Error
![image](https://user-images.githubusercontent.com/45851500/174605609-0467ac93-8fef-42da-a37c-ea0920936567.png)

### The save command
![image](https://user-images.githubusercontent.com/45851500/174607215-46dacfef-8924-4a3b-bbbd-22ff5fa4b679.png)

### The structure
![image](https://user-images.githubusercontent.com/45851500/174607002-04212c23-e1ac-408e-9576-f4ca8b3bada7.png)

### The generation of the "Auftragsnummer"
![image](https://user-images.githubusercontent.com/45851500/174606655-5a1a3e11-0819-4f55-ab34-593e6efe1477.png)

### and this is how my table looks like
![image](https://user-images.githubusercontent.com/45851500/174607761-4abff3c7-6a6c-440f-8e97-780d197ce71c.png)

**Comment by feurer98** [comment]
is this related to #6387 ?

**Comment by feurer98** [comment, marked as answer]
I have found my mistake.
The construction was wrong and I had a form in a form.

### passing on this experience:
Could we mark with colors or symbols what kind of element we are using?

Briefly illustrated with Photoshop (not all fields marked, but just briefly to illustrate)

![Github Budibase](https://user-images.githubusercontent.com/45851500/174645372-b8eddc0a-9dbd-41d0-b217-565e93d08396.png)

**Reply by melohagan** [reply]
When the upcoming new design is released, icons will be placed beside the dropdown arrows to make it clearer what components are being used in the tree :)
